charts/kueue/templates/manager/manager.yaml

```diff
@@ -67,6 +67,8 @@ spec:
         - mountPath: /tmp/k8s-webhook-server/serving-certs
           name: cert
           readOnly: true
+        - mountPath: /tmp
+          name: tmp
```

Comment:
It looks like this is a WIP PR.

/retitle WIP: add read only root filesystem for kueue deployment